recipe_scrapers/allrecipes.py

```diff
@@ -1,7 +1,19 @@
 from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
 
 
-class AllRecipes(AbstractScraper):
+class AllRecipes:
+    @classmethod
+    def host(cls):
+        return "allrecipes.com"
+
+    def __new__(cls, url, *args, **kwargs):
+        if AllRecipesUser.host() in url:
+            return AllRecipesUser(url, *args, **kwargs)
+        return AllRecipesCurated(url, *args, **kwargs)
```

Comment:
I like this solution to the URL-to-scraper mapping problem :+1: 

It keeps all the code for a single domain-name within the same Python module, which seems good logically -- while also providing distinct scraper classes for different page formats.

Since it's a rare use-case (so far, at least) it's also a benefit that it doesn't modify the existing host-to-scraper mapping and construction behaviour.